### PR TITLE
Edited the text regarding transport relays

### PR DESCRIPTION
**Pull request by FilippSe**
Edited the text regarding transport relays

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -74,13 +74,12 @@ The following diagram shows call flow when media bypass is enabled, the client i
 
 The following describes call flow if the user does not have access to the public IP address of the SBC. 
 
-For example, assume the user is external, and the tenant administrator decided not to open the public IP address of the SBC to everyone in the Internet, but only to the Microsoft Cloud. The internal components of traffic can flow via the Teams Transport Relays. This is the recommended configuration for users outside of the corporate network. Consider the following:
+For example, assume the user is external, and the tenant administrator decided not to open the public IP address of the SBC to everyone in the Internet, but only to the Microsoft Cloud. The internal components of traffic can flow via the Teams Transport Relays. Consider the following:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md:77]:
```suggestion
For example, assume the user is external, and the tenant administrator decided not to open the public IP address of the SBC to everyone on the internet, but only to the Microsoft cloud. The internal components of traffic can flow via the Teams Transport Relays. Consider the following:
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
Internet needs to be initial capped.  Please let the vendors know.

**Comment by JohanFreelancer9** [comment]
@CarolynRowe I would certainly agree with you that Internet should be initial capped, but for editing purposes, I am required to follow the Microsoft Style Guide, which states: "Use internet to refer to the worldwide collection of networks that use open protocols such as TCP/IP to communicate with one another. **Don't capitalize**. Your view please.

**Comment by CarolynRowe** [comment]
It would be inconsistent with all the other docs in this library. For now, please leave Internet initial capped. I’ll take responsibility for this decision.

Sent from my iPhone

On May 21, 2020, at 6:00 PM, JohanFreelancer9 <notifications@github.com> wrote:

﻿

@CarolynRowe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FCarolynRowe&data=02%7C01%7Ccrowe%40microsoft.com%7Cd3439f40ab42492fcb6608d7fdeb8da0%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637257060444466708&sdata=SxKl31syZP%2FlKOsPwG2c%2F09Jfxm1%2BMVDLWNX55p%2BOuY%3D&reserved=0> I would certainly agree with you that Internet should be initial capped, but for editing purposes, I am required to follow the Microsoft Style Guide, which states: "Use internet to refer to the worldwide collection of networks that use open protocols such as TCP/IP to communicate with one another. Don't capitalize. Your view please.

**Pull request merged**